build_projects/shared-build-targets-utils/Utils/Crossgen.cs

```diff
@@ -154,11 +195,8 @@ public void CrossgenDirectory(string sharedFxPath, string pathToAssemblies)
                     "-platform_assemblies_paths", platformAssembliesPaths
                 };
 
-                if (CurrentPlatform.IsUnix)
-                {
-                    crossgenArgs.Add("-JITPath");
-                    crossgenArgs.Add(GetLibCLRJitPathForVersion());
-                }
+                crossgenArgs.Add("-JITPath");
+                crossgenArgs.Add(GetLibCLRJitPathForVersion());
```

Comment:
I know this was existing code, but this method doesn't need to be called inside the 'foreach' loop.